### PR TITLE
[FEAT] std

### PR DESCRIPTION
**Pull request by MartonDev**
Deno std modules on nest.land

- [x] std route
  - [x] std view design
  - [x] fetch std module info
  - [x] parse std submodules
  - [x] auto-generate urls for std submodules
  - [x] add a file explorer component for std submodules with their respective paths
- [x] rewrite std f.a.q.

### CODE REVIEW

File: src/views/StdModule.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      //TODO: replace this to std before merge
+      await axios
+        .get(`https://x.nest.land/api/package/std-test/${ this.version }`)
+        .then(res => {
+          if(`/${ this.module }/README.md` in res.data.files) {
+            //TODO: replace this to std before merge
+            axios
+              .get(`https://x.nest.land/std-test@${ this.version }/${ this.module }/README.md`)
+              .then(readmeRes => {
+                //replace https://deno.land/std urls to https://x.nest.land/std
```
Comment (MartonDev) [inline src/views/StdModule.vue:79]: With this we'll be able to serve READMEs with our links instead of deno.land/std

**Comment by MartonDev** [comment]
TODO: fix markdown files

**Comment by MartonDev** [comment]
Ready for review

File: src/components/home/FAQ.vue [hunk 2]
```diff
@@ -53,7 +53,7 @@
                 <p
                   class="subtitle has-text-dark"
                 >Does nest.land have Deno's Standard Modules built in?</p>
-                <p>No, we do not. Nest.land is meant to work in tandem to Deno. Our goal is not to replace all of Deno's Standard Modules with our own or copy others' modules.</p>
+                <p>Yes, we do! If you wish to ensure security and long time support for your project, that uses the std library, nest.land provides an immutable <router-link to="/std">mirror</router-link> to it.</p>
```
Comment (t8) [inline src/components/home/FAQ.vue:56]:
```suggestion
                <p>Yes, we do! If you wish to ensure security and immutability of your standard modules, check out nest.land's std <router-link to="/std">mirror</router-link>.</p>
```

**Comment by MartonDev** [comment]
Bug on mobile: version select not responsive

**Comment by t8** [comment]
Last thing I have to say: It would be extremely beneficial if we also offer the Arweave import urls for these modules, as this url is truly immutable. Perhaps this could be added to the dropdown or tooltip on the way?

**Comment by MartonDev** [comment]
It's possible, I'm just not sure design-wise. I'll figure out something

**Comment by t8** [comment]
Happy to help with the design aspect of that.

**Comment by MartonDev** [comment]
That'd be awesome

**Pull request merged**